packages/core/src/definitions/styles.ts

```diff
@@ -56,6 +56,13 @@ export const builtInStyles: VariableDefinitions = {
     defaultValue: 'center',
     displayName: 'Horizontal alignment',
   },
+  cfVisibility: {
+    displayName: 'Visibility toggle',
+    type: 'Boolean',
+    group: 'style',
+    defaultValue: false,
```

Comment:
```suggestion
    defaultValue: true,
```
I thought nodes should be visible, per default